react-components/src/hooks/useModelIdRevisionIdFromModelOptions.test.tsx

```diff
@@ -64,53 +58,73 @@ describe(useModelIdRevisionIdFromModelOptions.name, () => {
   });
 
   test('returns modelId & revisionId for classic model option', async () => {
-    dependencies.useQueriedAddModelOptions.mockReturnValue([mockClassicModelResult]);
+    const resultMock = new Mock<UseQueryResult<Array<AddModelOptions<ClassicDataSourceType>>>>()
+      .setup((p) => p.data)
+      .returns([classicModelOption])
+      .setup((p) => p.isFetching)
+      .returns(false)
+      .setup((p) => p.isLoading)
+      .returns(false)
+      .setup((p) => p.isError)
+      .returns(false)
+      .setup((p) => p.isRefetching)
+      .returns(false)
```

Comment:
It seems to me that these fields are not necessary in the new implementation? (A few more occurences of this further down in this file as well)

I also think I'd prefer that the context of the hook could have a narrower type (something like `() => { data: Array<...> }`), so that we won't need `new Mock`, and can avoid all unnecessary fields